plugins/postcss-cascade-layers/src/sort-root-nodes.ts

```diff
@@ -70,22 +71,6 @@ export function sortRootNodes(root: Container, model: Model) {
 	});
 
 	root.nodes.sort((a, b) => {
-		const aIsCharset = a.type === 'atrule' && a.name.toLowerCase() === 'charset';
-		const bIsCharset = b.type === 'atrule' && b.name.toLowerCase() === 'charset';
-		if (aIsCharset && bIsCharset) {
-			return 0;
-		} else if (aIsCharset !== bIsCharset) {
-			return aIsCharset ? -1 : 1;
-		}
-
-		const aIsImport = a.type === 'atrule' && a.name.toLowerCase() === 'import';
-		const bIsImport = b.type === 'atrule' && b.name.toLowerCase() === 'import';
-		if (aIsImport && bIsImport) {
-			return 0;
-		} else if (aIsImport !== bIsImport) {
-			return aIsImport ? -1 : 1;
-		}
```

Comment:
All this code was not needed.
If users have incorrect orders for `@charset` or `@import` in their source we must leave it as it was, not try to fix it.